### PR TITLE
#184 Converted the classic pipeline to yaml Templates CI.

### PR DESCRIPTION
**Pull request by Harinder-accolite**
converted classic pipeline to yaml pipeline, details are below.

classic pipeline:-

repository: https://github.com/lansa/azure-quickstart-templates
branch: support/scalable
pipeline name - Template CI

converted classic to yaml pipeline
repository: Harinder-accolite:feature/harinder-classic2yaml
branch: feature/harinder-classic2yaml
pipeline name: Templates CI 3.0
pipeline url: https://dev.azure.com/VisualLansa/Lansa%20Azure%20Scalable%20License%20Images/_build/results?buildId=1800&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=275f1d19-1bd8-5591-b06b-07d489ea915a

1) Since the value 'Standard' is not supported and is deprecated, we have made changes in ApplicationGatewaySkuTier and the value is replaced with "Standard_v2".
2) Because of that change, agpublicIpSku does not support Basic in SKU, so changed it to "Standard".
3) Also because of above changes, applicationgatewayskuName was changed to to 'standard_v2'.
4) agpulicIpAllocationMethod was changed to Static, because applicationgatewayskuName was changed to standard_v2 (the step above).

### CODE REVIEW

**Review by robe070 — CHANGES_REQUESTED** [review]
Name the yml file the same as the pipeline UI name, but without the version number - "3.0"
"Templates - AWS CI"

**Comment by Harinder-accolite** [comment]
This yaml Template CI pipeline (https://dev.azure.com/VisualLansa/Lansa%20Azure%20Scalable%20License%20Images/_build/results?buildId=1800&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=275f1d19-1bd8-5591-b06b-07d489ea915a) belongs to Azure do you still want me to mention AWS.

File: Template CI.yaml [hunk 1]
```diff
@@ -0,0 +1,333 @@
+name: Template-CI-3-$(Build.BuildId)
```
Comment (robe070) [inline Template CI.yaml]: Take the 3 out of here too.

**Review by robe070 — CHANGES_REQUESTED** [review]
The filename, as I've said before, must include the cloud too, if the pipeline name does not already include it.

So "Azure Template CI.yml"

File: Template CI.yaml (hunk 1 above)
Comment (Harinder-accolite) [reply Template CI.yaml]: Renamed the "Template-CI-3-$(Build.BuildId)" to Template-CI-$(Build.BuildId)

**Comment by Harinder-accolite** [comment]
Changes made as requested

> The filename, has been changed to "Azure Template CI .yaml"

**Comment by robe070** [comment]
Now I've merged the commit you need to change the default repo and branch back in the UI and variables to be the LANSA one(s). Then re-run the pipeline running all jobs in one run.

**Pull request merged**